core/capabilities/integration_tests/keystone/contracts_setup.go

```diff
@@ -48,38 +47,3 @@ func SetupConsumerContract(t *testing.T, backend *framework.EthBlockchain,
 
 	return addr, consumer
 }
-
-// secureMintFeedDataID is the Data ID generated for a secure mint feed using the feed_id.go generator in CLD
-const secureMintFeedDataID = "0x01c508f42b0201320000000000000000"
-
-func SetupDataFeedsCacheContract(t *testing.T, backend *framework.EthBlockchain,
-	forwarderAddress common.Address, workflowOwner string, workflowName string) (common.Address, *data_feeds_cache.DataFeedsCache) {
-	addr, _, dataFeedsCache, err := data_feeds_cache.DeployDataFeedsCache(backend.TransactionOpts(), backend.Client())
-	require.NoError(t, err)
-	backend.Commit()
-
-	var nameBytes [10]byte
-	copy(nameBytes[:], workflowName)
-
-	ownerAddr := common.HexToAddress(workflowOwner)
-
-	_, err = dataFeedsCache.SetFeedAdmin(backend.TransactionOpts(), backend.TransactionOpts().From, true)
-	require.NoError(t, err)
-	backend.Commit()
-
-	feedIDBytes := [16]byte{}
-	copy(feedIDBytes[:], common.FromHex(secureMintFeedDataID))
-
-	_, err = dataFeedsCache.SetDecimalFeedConfigs(backend.TransactionOpts(), [][16]byte{feedIDBytes}, []string{"securemint"},
-		[]data_feeds_cache.DataFeedsCacheWorkflowMetadata{
-			{
-				AllowedSender:        forwarderAddress,
-				AllowedWorkflowOwner: ownerAddr,
-				AllowedWorkflowName:  nameBytes,
-			},
-		})
-	require.NoError(t, err)
-	backend.Commit()
-
-	return addr, dataFeedsCache
-}
```

Comment:
I wonder if we need to be careful about deleting exported code, even though the only use will also be deleted. Consider if the last release of chainlink/v2 needs a new patch version, with an update of chainlink-common. In that scenario, any old code referencing this would break. WDYT about swapping in a skip?
```go
func SetupDataFeedsCacheContract(t *testing.T, backend *framework.EthBlockchain,
	forwarderAddress common.Address, workflowOwner string, workflowName string) (common.Address, *data_feeds_cache.DataFeedsCache) {
	t.Skip("No longer supported")
}
```